src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/ViewHelper.java

```diff
@@ -94,4 +105,37 @@ public static void set(
         setPivotXIfNeeded(view, pivotX);
         setPivotYIfNeeded(view, pivotY);
     }
+
+    @NonNull
+    public static LinearLayout createNavigationBarOuterLayout(Context context)
+    {
+        LinearLayout linearLayout = new LinearLayout(context);
+        linearLayout.setOrientation(LinearLayout.VERTICAL);
+        linearLayout.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT));
+        return linearLayout;
+    }
+
+    @NonNull
+    public static FrameLayout createNavigationBarArea(Context context, LinearLayout linearLayout)
+    {
+        FrameLayout frameLayout = new FrameLayout(context);
+        frameLayout.setId(View.generateViewId());
+        LinearLayout.LayoutParams layoutParams = new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, 0);
+        layoutParams.gravity = Gravity.FILL;
+        layoutParams.weight = 1;
+        frameLayout.setLayoutParams(layoutParams);
+        linearLayout.addView(frameLayout);
+        return frameLayout;
+    }
+
+    @NonNull
+    public static BottomNavigationView createNavigationBar(Context context, int navigationStyle, LinearLayout linearLayout, BottomNavigationView.OnItemSelectedListener listener)
+    {
+        BottomNavigationView navigationView = new BottomNavigationView(new ContextThemeWrapper(context, navigationStyle));
+        navigationView.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT));
```

Comment:
Can the style get applied via an attribute so we don't have to use a `ContextThemeWrapper`? 

https://github.com/xamarin/Xamarin.Forms/search?q=materialProgressBarHorizontalStyle

Reply:
Yes, it seems to work this way.

So I added to `Maui.MainTheme`:
```xml
<item name="bottomNavigationViewStyle">@style/Widget.Design.BottomNavigationView</item>
```